core/src/env_platform/utility.bal

```diff
@@ -48,3 +48,39 @@ function extractTreeInformationAsJSONArray(TreeInformation[] treeInfoArray) retu
     }
     return treeInformation;
 }
+
+# The `constructApplicationVersion` function will construct the application version which will be suitable for the database.
+# 
+# + form - TreeRemovalForm type record.
+# + return - Returns the map<json> which suites for the database
+function constructApplicationVersion(TreeRemovalForm form) returns map<json> {
```

Comment:
Will the name `constructApplicationVersion` be suitable since we use this to perform altering if application is a draft and versioning if application is already submitted? @sanjulamadurapperuma @yashodgayashan thoughts?

Reply:
Should it be constructApplication or?

Reply:
Yes I missed this, it has to be generic

Reply:
Yeah it should be general like `constructApplication`.

Reply:
Any suggestions?

Reply:
`constructApplication` is good enough i guess.

Reply:
Yes. It is